scripts/run_integration_tests.sh

```diff
@@ -40,46 +42,67 @@ else
     echo "Not building layers, ensure they've already been built or re-run with 'BUILD_LAYERS=true DD_API_KEY=XXXX ./scripts/run_integration_tests.sh'"
 fi
 
+# Add local build to node_modules so `serverless-plugin.yml` also has access to local build.
+yarn build
+rm -rf "$integration_tests/node_modules"
+mkdir -p "$integration_tests_dir/node_modules/datadog-lambda-js"
+cp -r dist "$integration_tests_dir/node_modules/datadog-lambda-js"
```

Comment:
We need to do this in order to use the local code since the plugin test is using `addLayers:false`